timeline-chart/src/layer/time-graph-navigator.ts

```diff
@@ -80,7 +80,8 @@ export class TimeGraphNavigatorHandle extends TimeGraphComponent {
 
     render(): void {
         const position = { x: this.unitController.viewRange.start * this.stateController.absoluteResolution, y: 0 };
-        const width = this.unitController.viewRangeLength * this.stateController.absoluteResolution;
+        // Set min width to 20 to make it appear at all zoom levels
+        const width = Math.max(20, this.unitController.viewRangeLength * this.stateController.absoluteResolution);
```

Comment:
Perhaps extract the value `20` into a constant, and then the comment isn't required?

Reply:
updated.